### PR TITLE
Math Magicians: Event

### PR DESCRIPTION
**Pull request by chiedozie013**
## In this branch, I:
- [ ] Create the logic directory inside the project,
- [ ] Copy the files calculate.js and operate.js into your logic/ directory,
- [ ] Install big.js using npm
- [ ] Import calculate.js in your Calculator.js component,
- [ ] Implement click event handlers using the math logic from calculate.js and operate.js in my React component, the goal is to make all calculator buttons work according to their functionality e.g. the - button should subtract a given value,
- [ ] Test my app in the browser. The Calculator is able to perform math operations.
- [ ] Ensure no linters error

### CODE REVIEW

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @chiedozie013 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  Correct operate.js and calculate.js files

✔️ Calculator works correctly



## Required Changes ♻️
- [ ] Kindly make sure that in your   child component  props are passed correctly  from the Calculator.js component, this will help you practice   what you have learned in `props`, currently you have a child component but no props are received from the Calculator component👍🏿 
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/calculator.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    return {};
+  }
+
+  // Button must be an operation
+
+  // When the user presses an operation button without having entered
+  // a number first, do nothing.
+  // if (!obj.next && !obj.total) {
+  //   return {};
+  // }
```
Comment (t-yanick) [inline src/logic/calculator.js]: - [ OPTIONAL] Kindly remove all pieces of commented codes from your codebase. This makes your codebase look clean and professional.

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @chiedozie013 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Requested change implemented

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/calculator.js (hunk 1 above)
Comment (chiedozie013) [reply src/logic/calculator.js]: Thanks for pointing that out. I appreciate

**Pull request merged**